sessions/react-effects-and-fetch/pokemon-api/README.md

```diff
@@ -0,0 +1,34 @@
+# React Effects and Fetch: Pokémon API
+
+This app should load and display a list of Pokémon.
+
+## Task
+
+Open the file [`src/components/PokemonList.js`](./src/components/PokemonList.js).
+
+For now the Pokémon are only fetched when we click the button below. Lets change the code such our app does this automatically when the app renders.
```

Comment:
```suggestion
For now the Pokémon are only fetched when we click the button below. Let's change the code so that our app does this automatically when the app renders.
```